Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -475,10 +476,35 @@
   components:
   - type: Sprite
     state: butter
+  - type: Slippery
+    paralyzeTime: 1.5
+    launchForwardsMultiplier: 1.2
```

Comment:
You might want to adjust the launchForwardMultiplier to 1.5 so it is in accordance with the new standardised slip stats #27879

Reply:
Oh yea can do. I intended for it to be less good than soap and more of a kitchen drop hazard when soap was stronger, but I'm not so fussed about it. I'll just yoink the paralyzeTime and launchForwardMultiplier fields from being set.